code/network/multi_observer.cpp

```diff
@@ -60,6 +60,7 @@ void multi_obs_create_player(int player_num,char *name,net_addr *addr,player *pl
 	Net_players[player_num].reliable_socket = PSNET_INVALID_SOCKET;
 	Net_players[player_num].s_info.kick_timestamp = -1;
 	Net_players[player_num].s_info.voice_token_timestamp = -1;
+	Net_players[player_num].s_info.player_collision_timestamp = -1;
```

Comment:
This should be initialized to `timestamp(-1)`

(and almost certainly `kick_timestamp` and `voice_token_timestamp` as well)

Reply:
Changes made

Reply:
I switched it back to `timestamp(0)` because there's no need to set it to an invalid timestamp and check for invalid timestamp every time there's a collision.  It seems like timestamp(0) was set up for this exact situation.